### ISSUE TITLE
arkd should return a history of transactions 

### ISSUE THREAD

**Issue by bordalix** [ark-network/ark#263]

**Comment by tiero** [comment]
also this seems sdk job

**Comment by bordalix** [comment]
> to give more context @bordalix what are the data that you DONT have inside ListVtxos RPC and we have to cache it for the user?

Like done in the PWA wallet, from the ListVtxos I think I manage to reconstruct all transactions for the user.
Having said that, the UI mock up shows a **failed** tx, which will not be possible to find out using this strategy. 

> also this seems sdk job

Yes, it's odd ListVtxos is not implemented on the SDK

**Comment by tiero** [comment]
Yes indeed I think beside the VTXO data we may have extra meta-data.

For the failed payment I think that also would depend if we keep that historical information: failed should be a payment is sent but recipient never claimed. Guess in practice we loose this information, unless ASP stores it
